src/main/webapp/i18n/de/faq.json

```diff
@@ -4,12 +4,19 @@
             "home": {
                 "title": "FAQ",
                 "createLabel": "FAQ erstellen",
+                "proposeLabel": "FAQ vorschlagen",
+                "accept": "FAQ akzeptieren",
+                "reject": "FAQ ablehnen",
                 "filterLabel": "Filter",
                 "createOrEditLabel": "FAQ erstellen oder bearbeiten"
             },
-            "created": "Das FAQ wurde erfolgreich erstellt",
-            "updated": "Das FAQ wurde erfolgreich aktualisiert",
-            "deleted": "Das FAQ wurde erfolgreich gelöscht",
+            "created": "Das FAQ mit der ID {{ id }} wurde erfolgreich erstellt",
```

Comment:
"Das" oder "die" Nutella?

Reply:
Good question. I Actually dont care whether we use Das or Der. What do you prefer?

Reply:
<img width="710" alt="image" src="https://github.com/user-attachments/assets/d85324bf-7067-4743-afe9-27d11ba17b55">

In case my short google research is correct I think that the sentence is in nominativ and therefore `Die` would be correct?
However, I might be mistaken and have no strong opinion.